sss/sss_table.py

```diff
@@ -54,59 +143,120 @@ class SSS(Base):
     annual_self_sufficiency_wage = Column(
         'annual_self_sufficiency_wage', Float)
     emergency_savings = Column('emergency_savings', Float)
-    
     miscellaneous_is_secondary = Column(
         'miscellaneous_is_secondary', Boolean)
     health_care_is_secondary = Column(
         'health_care_is_secondary', Boolean)
-
     analysis_is_secondary = Column(
         'analysis_is_secondary', Boolean)
+
+class ARPA(Base):
+    """
+    state : str column 
+        One of our primary keys, which what state this data is taken from.
+
+    place : str
+        One of our primary keys, which either the town or county where the data was collected. 
+
+    year : int column 
+        One of our primary keys, the year the data is from. 
+
+    analysis_type : str column 
+        One of our primary keys, whether the file is either a town or county.
+
+    federal_state_eitc : float column
+        Amount of the "refundable" tax credit based on loss of income. 
+    
+    federal_cdctc : float column 
+        Amount of Child and Dependent Care Tax Credit.
+
+    federal_income_taxes : float column
+        Amount of federal income taxes paid.
+    
+    payroll_taxes : float column
+        Amount of payroll taxes paid.
     
+    state_sales_taxes : float column 
+        Amount of state sales taxes paid. 
+
+    total_annual_resources : float column
+        An adjusted annual wage to reach self-sufficiency. 
+    """
+    __tablename__ = 'arpa'
+    family_type = Column('family_type', String, primary_key = True)
+    state = Column('state', String, primary_key = True)
+    place = Column('place', String, primary_key = True)
+    year = Column('year', Integer, primary_key = True)
+    analysis_type =  Column('analysis_type', String, primary_key = True)
+    federal_and_oregon_eitc = Column('federal_and_oregon_eitc', Float)
+    # federal_oregon_eitc = Column('federal_state_eitc', Float)
+    # federal_state_eitc = Column('federal__eitc', Float)
```

Comment:
we should remove these commented lines. Also, I thought we decided to go with "federal_state" rather than "federal_oregon" but I'm not sure about that, we should check with @cwwannie